parsec/backend/asgi/rpc.py

```diff
@@ -84,14 +85,27 @@ def _rpc_msgpack_rep(data: dict[str, object], api_version: ApiVersion) -> Respon
 # - 461: User is revoked
 
 
+class CustomHttpStatus(Enum):
+    BadAuthenticationInfo = 401
+    OrganizationOrInvitationInvalidOrNotFound = 404
+    BadAcceptType = 406
+    InvitationAlreadyUsedOrDeleted = 410
+    BadContentTypeOrInvalidBodyOrUnknownCommand = 415
+    UnsupportedApiVersion = 422
+    OrganizationExpired = 460
+    UserRevoked = 461
```

Comment:
Should I use constant instead of an enum ?

Reply:
I think enum like this is fine :-)